pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -212,6 +214,8 @@ func TestUpdateNewNodeStatus(t *testing.T) {
 			defer testKubelet.Cleanup()
 			kubelet := testKubelet.kubelet
 			kubelet.nodeStatusMaxImages = tc.nodeStatusMaxImages
+			kubelet.kubeletConfiguration.MemorySwap.SwapBehavior = types.NoSwap
+			isSwapOn, _ := swap.IsSwapOn()
```

Comment:
We don't need to get the real value in unit test? Here we can just define it as `false`

Reply:
The previous commit presents a thorough set of unit tests. In this commit I'm only adapting the existing unit tests.

> Here we can just define it as false

the condition would show the real value, which can be different between environments. If there's a need, I can make changes so that we could inject a function to check if swap is on, then make a mock here, but TBH I'm not sure what's the added value in doing so. I think it will complicate the code with no real test value. As said above, my whole intention here is for the test to be adapted to the new changes. The unit tests that are actually meant for testing swap exist in the previous commit.

Reply:
@iholder101 has a point, but OTOH by injecting a fake function we can test the full range of expecting conditions without depending on the current system properties, which is quite often fragile and more complex in the long run. I'm not pushing for this change (again Itamar has a point), but I'd really prefer a fully controlled unit test environment. Same reason we mock `time.Now` after all.

Reply:
> If there's a need, I can make changes so that we could inject a function to check if swap is on

Yes, that's what we should do. I'm just worried that if `IsSwapOn()` returns an error, the operation results will be inconsistent with expectations.

Reply:
Thanks @ffromani.
I fully agree that we should strive for a very high test coverage and a fully controlled unit test environment, I just believe I've covered that in [`02cbc93` (#125278)](https://github.com/kubernetes/kubernetes/pull/125278/commits/02cbc937755951f0534777f0f59e4f489fc89da6). There, I'm injecting a mock function (since the setter is being called directly) and test the full range of expected conditions.

WDYT?

Reply:
> I'm just worried that if IsSwapOn() returns an error,

Fair point.
Okay, I'll make the changes.